install/020-ConfigMap.yaml

```diff
@@ -29,4 +29,4 @@ data:
     logger.bridge.level=INFO
     logger.bridge.appenderRefs=console
     logger.bridge.appenderRef.console.ref=STDOUT
-    logger.bridge.additivity=false
\ No newline at end of file
+    logger.bridge.additivity=false
```

Comment:
this file should not have any difference, just revert the change. I see you deleted the blank line. Changing it just for a blank line would be confusing because not related to the PR itself.